# Generated by Django 5.0.3 on 2024-03-28 19:07

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('students', '0009_delete_extra_booking_delete_mealreservation'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='student',
            name='remaining_fees',
        ),
        migrations.AddField(
            model_name='student',
            name='rollno',
            field=models.CharField(default=0, max_length=15, unique=True),
            preserve_default=False,
        ),
        migrations.AlterField(
            model_name='student',
            name='email',
            field=models.EmailField(max_length=254),
        ),
        migrations.AlterField(
            model_name='student',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
